CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# TODO(CK): set(CMAKE_CXX_MODULE_STD 1)
+if(CMAKE_CXX_MODULE_STD)
+    set(CMAKE_EXPERIMENTAL_CXX_IMPORT_STD
+        "d0edc3af-4c50-42ea-a356-e2862fe7a444"
+    )
+    set(CMAKE_CXX_STANDARD 26)
```

Comment:
In discussions at c++now about this we made the decision that we weren't setting the standard in our cmake so as to not override the user options.  If the library requires a certain level well that needs to be check in header and cause compile failure.  That's the better approach since a user might just lift this code and not use our cmake at all